pennylane/transforms/qcut.py

```diff
@@ -424,7 +441,7 @@ def _get_measurements(
 
     obs = measurement.obs
 
-    return [expval(obs @ g) for g in group]
+    return [expval(copy.deepcopy(obs) @ g) for g in group]
```

Comment:
I am very confused by this. the tensor product works in-place :thinking: Perhaps it is a queuing-related quirk.

```python
obs = qml.PauliZ(0) @ qml.PauliZ(1)
obs2 = obs @ qml.PauliZ(2)
assert obs == obs2
print(obs)
```

Reply:
Also works with `copy.copy` so I changed to that. I think `copy.copy` is a sufficient workaround for now, but long run it'd be better for observables not to be altered when taking the tensor product. Issue posted: https://github.com/PennyLaneAI/pennylane/issues/2235